modular_zubbers/modules/customization/modules/mob/living/carbon/human/species/tajaran.dm

```diff
@@ -0,0 +1,79 @@
+//Tajaran Bubber edit
+//makes them more like Citrp's tajara aka snow cats
+
+/datum/species/tajaran
+	inherent_traits = list(
+		TRAIT_CATLIKE_GRACE,
+	)
```

Comment:
soooo this breaks coloring I'm right now trying to fix this if my fix is not allowed no cat like grace then sadly